internal/deprecated/control-plane.go

```diff
@@ -44,11 +45,12 @@ func RunControlPlane(config config.Mock, k8s kubernetes.Interface, i *integratio
 		return err
 	}
 
-	nodeName := os.Getenv(nodeNameEnvVar)
-	if nodeName == "" {
-		logger.Errorf("%s env var should be provided by Kubernetes and is mandatory", nodeNameEnvVar)
-		os.Exit(1)
-	}
+	nodeName := config.NodeName
+	// nodeName := os.Getenv(nodeNameEnvVar)
```

Comment:
why commenting out and not deleting? (however I guess this disappear in the next commit)

Reply:
I deleted this file actually, this will be vanished after rebase